internal/gcs/protocol.go

```diff
@@ -0,0 +1,261 @@
+package gcs
```

Comment:
We should move the OpenGCS to a binary in hcsshim. It would be nice to share this but not have to make it a public package.

Reply:
Yeah.